README_kor.md

```diff
@@ -0,0 +1,47 @@
+![MagicWEv2의 널찍하고 아름다운 현수막!](https://github.com/thebigsmileXD/MagicWE2/blob/master/resources/magicwe_icon_wide.png)
+# MagicWEv2 - 강화되었고, 더 나아졌으며, 더 빠름과 동시에 더욱 강해짐
+더 많은 지원, 더 많은 명령어, 새로운 도구 등으로 더욱 강력해진 MagicWE!
+
+# 새로운 점
+말 그대로 모든 것이 새로워졌습니다.
+
+이 코드는 MagicWEv1의 지저분함과는 완전히 다릅니다. MagicWEv2에는 전체 플러그인에 새로운 모습에다가 무지개색 스프링클스 토핑을 추가했답니다.
+
+무지개 스프링클스를 사랑하지 *않는* 분 계신가요? 희한하네요!
+
+- 아이콘
+- 새로운 명령어
+- 새로운 기능
+- 번역
+- ***엄청난*** 속도
+- Windows 98 같은 진행 표시줄!
+- 세션
+- 클립보드
+- 개략도 지원
+- MyPlot 지원
+
+## 명령어
+- //wand - 이제 절대로 작동을 멈추지 않는 지팡이 도구를 지급하며, 일반 나무 도끼는 원래대로 작동합니다.
```

Comment:
@thebigsmileXD So apparently I added a period to the end of the string... Can you remove it? I don't want to create new pull request because it looks spammy IMO.

Reply:
Which exact symbol is that?

Reply:
@Johnmacrocraft It's alright, the period can exist there even though it isn't shown in the English version.

Reply:
@Aericio Of course it can be, but the reason why I'm asking that is I don't like adding these punctuation marks if they don't exist on the original - that's the rule of Minecraft: Java Edition Korean translations, and as a part of MCJE translators, I agree and follow the rules.

Reply:
If you mind than you don't need to do that, I'll do it later if I need to make some other changes.